virt/virt-4-11-release-notes.adoc

```diff
@@ -105,6 +105,15 @@ Deprecated features are included in the current release and supported. However,
 Removed features are not supported in the current release.
 
 //CNV-16317 NMState is no longer part of CNV
+* {VirtProductName} {VirtVersion} removes support for link:https://nmstate.io/[nmstate], including the following objects:
++
+--
+** `NodeNetworkState`
+** `NodeNetworkConfigurationPolicy`
+** `NodeNetworkConfigurationEnactment`
+--
++
+Instead, you can use nmstate features by installing the xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Kubernetes NMState Operator] before updating to {VirtProductName} {VirtVersion}. You can install it from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`). The Kubernetes NMState Operator preserves and supports the existing nmstate configuration.
```

Comment:
I wonder if users will find the phrase "Instead, you can use nmstate features by..." a bit unclear re: support implications. What about something like this?

```suggestion
To preserve and support your existing nmstate configuration, install the xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Kubernetes NMState Operator] before updating to {VirtProductName} {VirtVersion}. You can install it from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`). 
```